libs/partners/xai/langchain_xai/chat_models.py

```diff
@@ -266,7 +266,6 @@ class Joke(BaseModel):
     """Base URL path for API requests."""
 
     openai_api_key: Optional[SecretStr] = None
-    openai_api_base: Optional[str] = None
```

Comment:
Why are we deleting public attributes?

Reply:
Because the integration has already defined an xai_api_base and this class  inherites from langchain_openai integration  which actually defined enough public attributes.